seed/challenges/02-javascript-algorithms-and-data-structures/basic-data-structures.json

```diff
@@ -345,7 +386,7 @@
       ],
```

Comment:
@dhcodes haha, no worries, nice catch yet again, better to get this right! All set